Source/Core/Core/PowerPC/SignatureDB.cpp

```diff
@@ -48,6 +51,59 @@ bool SignatureDB::Load(const std::string& filename)
   return true;
 }
 
+bool SignatureDB::LoadAsCSV(const std::string& filename)
+{
+  std::string line;
+  std::ifstream ifs;
+  OpenFStream(ifs, filename, std::ios_base::in);
+
+  if (!ifs)
+    return false;
+  for (size_t i = 1; std::getline(ifs, line); i += 1)
+  {
+    std::istringstream iss(line);
+    u32 checksum, size;
+    std::string tab, symbol, object_location, object_name;
+
+    iss >> std::hex >> checksum >> std::hex >> size;
+    if (iss && !std::getline(iss, tab, '\t') || !std::getline(iss, symbol, '\t') ||
+        !std::getline(iss, object_location, '\t') || std::getline(iss, object_name))
+    {
+      DBFunc dbf;
+      dbf.name = symbol;
+      dbf.size = size;
+      database[checksum] = dbf;
+    }
+    else
+    {
+      WARN_LOG(OSHLE, "CSV database failed to parse line %d", i);
+    }
+  }
+
+  return true;
+}
+
+// CSV separated with tabs
+// Checksum | Size | Symbol | [Object Location |] Object Name
+bool SignatureDB::SaveAsCSV(const std::string& filename)
+{
+  File::IOFile f(filename, "wb");
+
+  if (!f)
+  {
+    ERROR_LOG(OSHLE, "CSV database save failed");
+    return false;
+  }
+  for (const auto& func : database)
+  {
+    fprintf(f.GetHandle(), "%08x\t%08x\t%s\t%s\t%s\n", func.first, func.second.size,
+            func.second.name.c_str(), "", "");
```

Comment:
You may want to leave a comment that the last two strings are currently unimplemented/to be implemented